TAF/utils/scripts/docker/get-compose-file.sh

```diff
@@ -7,39 +7,36 @@ USE_SECURITY=${3:--}
 USE_RELEASE=${4:-nightly-build}
 
 # # x86_64 or arm64
-[ "$USE_ARCH" = "arm64" ] && USE_ARM64="-arm64"
+[ "$USE_ARCH" = "arm64" ] && USE_ARM64="-arm64" && ARM64_OPTION="arm64"
 
 # # security or no security
-[ "$USE_SECURITY" != '-security-' ] && USE_NO_SECURITY="-no-secty"
+[ "$USE_SECURITY" != 'security' ] && USE_NO_SECURITY="-no-secty" && NO_SECTY_OPTION="no-secty"
```

Comment:
USE_SECURITY value should be '-security-'.

Reply:
Opps, was play around with values and forgot to put that back.. THX!

Reply:
Fixed